app/config/tstvhq/rooms.json

```diff
@@ -261,13 +258,14 @@
         {
             "id": "sleepingPorch",
             "name": "Sleeping Porch",
-            "capacity": 10,
-            "description": "Check out these good kids! Sponsor one at woodstocksanctuary.org",
+            "capacity": 100,
+            "muteRoom": true,
+            "hideVideo": false,  
+            "description": "Hang out and chill",
             "type": "IFRAME",
             "backgroundImage": "./https://tstvhq.s3.amazonaws.com/backgrounds/sleeping porch.jpg",
             "iframeOptions": {
-                "src": "https://www.youtube.com/embed/_UItI96xdm4"
-            },
```

Comment:
this brace shouldn't be deleted, it needs to be below the "src" line

Reply:
'k